src/operator/account/AccountViewHeader.tsx

```diff
@@ -59,21 +61,26 @@ const AccountViewHeader: FC = () => {
           Convert to Contract
         </ButtonBase>
       )}
-      {hasWritePermissions && (
+      {hasWritePermissions && account?.deletable && (
         <ButtonBase
           color={ComponentColor.Danger}
           shape={ButtonShape.Default}
           onClick={() => setDeleteOverlayVisible(!deleteOverlayVisible)}
-          status={
```

Comment:
Isn't this a one-way operation - if you click the button, the visibility state always sets to `true` (instead of being toggled)?